### DISCUSSION TITLE
Noise description in JOSS paper

### DISCUSSION THREAD

**Discussion by beckynevin** [deepskies/DeepBench#73] [General]
@bnord I'm also wondering about if we want to go into a little more depth about the noise model in the JOSS paper; currently, it seems to be an aspect we talk about in the section that describes each individual module. But, I think we could highlight the noise model a little more if we wanted to also focus on the SBI aspect of this all.

The noise model is more complex for the pendulum since we're adding the capability to add different noise modifications to each individual parameter (as opposed to just adding noise like before, to the final image or object.

**Comment by bnord** [comment]
1. Is the pendulum noise something that can be abstracted to use with other physics objects?
2. I think relating the noise module to downstream usages is appropriate: i.e., through the statement of need.
